.enb/lib/page.js

```diff
@@ -0,0 +1,126 @@
+var techs = require('../techs');
+
+module.exports = function(node, options) {
+    var levels = options.levels || [],
+        browsers = options.browser || [],
+        langs = options.langs || [],
+        templateEngine = options.templateEngine || 'BH';
+
+    // essential
+    node.addTechs([
+        [techs.bem.levels, { levels : levels }],
+        [techs.bem.bemjsonToBemdecl],
+        [techs.bem.depsOld],
+        [techs.bem.files]
+    ]);
+
+    // css
+    node.addTechs([
+        [techs.css.stylusWithAutoprefixer, { browsers : browsers }],
+        [techs.css.stylus, { target : '?.ie.css', sourceSuffixes : ['styl', 'ie.styl'] }],
+    ]);
+    node.mode('development', function() {
+        node.addTechs([
+            [techs.borschik, { source : '?.css', target : '_?.css', freeze : true, minify : false }],
+            [techs.borschik, { source : '?.ie.css', target : '_?.ie.css', freeze : true, minify : false }]
+        ]);
+    });
+    node.mode('production', function() {
+        node.addTechs([
+            [techs.borschik, { source : '?.css', target : '_?.css', freeze : true, tech : 'cleancss', minify : true }],
+            [techs.borschik, { source : '?.ie.css', target : '_?.ie.css', freeze : true, tech : 'cleancss', minify : true }],
+        ]);
+    });
+    node.addTargets(['_?.css', '_?.ie.css']);
+
+    // JavaScript for browsers
```

Comment:
> let's create special helper for depsByTech which encapsulates bemdecl -> deps -> files steps:

@tadatuta it is a good idea. But I now think that we don't need client-side template engines for browser JavaScript.

@zxqfox, I understand your pain, but for this we need to change something in the current tool. This PR does not involve such changes.